lib/daru/index.rb

```diff
@@ -96,6 +96,27 @@ def [](*key)
         v
       end
     end
+    
+    def pos *args
+      # Causes Segmentation fault in array_wrapper
+      if args.first.is_a? Range
+        return args.first.to_a
```

Comment:
Can you also open an issue in nmatrix about this? Errors like these should be happening.